site/layouts/partials/header.html

```diff
@@ -6,7 +6,7 @@
         </a>
         <nav class="site-nav" role="navigation">
         <div>
-            <h1>&nbsp;<a href="/">ZAP</a></h1>
+            <h1>&nbsp;<a href="/"><span class="zapLogo">ZAP</span></a></h1>
```

Comment:
Was this a change from the other PR? I don't see this class defined anywhere and the header should not need to be changed for the footer changes.

Reply:
This change has been reverted.